src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -552,7 +546,7 @@ public static function provideSpecialCharacterUrl()
         ];
     }
 
-    protected function createMedia($id)
+    protected static function createMedia(int $id): Media
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes